torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -72,18 +72,20 @@ double getScaleFromInput(Node* input_node) {
   }
   // For the ops below the scale is not part of the op signature, so we traverse
   // up the graph to get the scale from its input when defined in the graph.
-  else if (input_name == "quantized::max_pool2d") {
-    auto tmp = input_node->inputs()[0]->node();
-    return getScaleFromInput(tmp);
-  } else if (input_name == "aten::relu") {
-    auto tmp = input_node->inputs()[0]->node();
-    return getScaleFromInput(tmp);
-  } else if (input_name == "prim::ListUnpack") {
-    return getScaleFromInput(input_node->inputs()[0]->node());
-  } else if (input_name == "aten::split_with_sizes") {
+  else if (
+      input_name == "quantized::max_pool2d" || input_name == "aten::relu" ||
```

Comment:
Can we create a white list (i.e., unordered_set) and check input_name exists in the white list or not?